src/renderer/scss/_gui.scss

```diff
@@ -1,86 +1,52 @@
-@import url(https://fonts.googleapis.com/css?family=Roboto:400,400i,500,500i,700);
+// Generic html styles used accross the App
+// component specific styling should go in the component scss file
 
-html {
-  height: 100%;
-  font-size: var(--font-size);
+// The actual fonts used will change ex: medium vs regular
+@font-face {
+  font-family: 'Metropolis';
+  font-weight: normal;
+  font-style: normal;
+  text-rendering: optimizeLegibility;
+  src: url('../../../static/font/metropolis/Metropolis-Medium.woff2') format('woff2');
```

Comment:
I think we are packaging some Metropolis font files we don't actually use. Intentional?

Reply:
I figured would just keep them all in for now so I didn't have to copy them in as I test them. We won't end using a lot of these.